application/api/messages.go

```diff
@@ -0,0 +1,37 @@
+package api
+
+import (
+	"time"
+
+	"github.com/gofrs/uuid"
+)
+
+// swagger:model
+type Messages []Message
+
+// A Message is one element associated with a conversation thread
```

Comment:
You could pull the docs from the graphql schema. I think they're a little more descriptive.

Reply:
You mean this?

`Return a specific message. If the message is not visible to the auth user, an error will be returned.`